README.md

```diff
@@ -58,6 +60,15 @@ The config file has the following sections:
   * [githubWorkflow\.license](#githubworkflowlicense)
   * [githubWorkflow\.spellCheck](#githubworkflowspellcheck)
 
+### `metadata`
+
+```yaml
+metadata:
+  url: https://github.com/sapcc/go-makefile-maker
```

Comment:
Showing go-makefile-maker as an example here is potentially confusing since the README is meant to be consumed by users of go-makefile-maker.
```suggestion
  url: https://github.com/foo/bar
```